datamonster_api/tests/lib/regression.py

```diff
@@ -43,10 +43,13 @@ def test_company():
     assert company.name == "AMAZON"
     assert company.ticker == "AMZN"
     assert company.type == "Company"
+    assert company.quarters
+    assert type(company.quarters) == list
+
     amzn_data_sources = set(company.datasources)
     amzn_data_source = dm.get_datasource_by_id("cd924848-5c49-4622-95a7-ee6d2cfe24b7")
     assert amzn_data_source.name in {i.name for i in amzn_data_sources}
-    assert len(amzn_data_sources) == 241
+    assert len(amzn_data_sources) == 263
```

Comment:
These seem really fragile. We'll probably want to swap them out for the QA ETL once it's running.

Reply:
yes, i am waiting for that!!

Reply:
it's actually hitting oasis in these tests?